### DISCUSSION TITLE
OBS studio does not work

### DISCUSSION THREAD

**Discussion by devansh-srv** [prasanthrangan/hyprdots#1910] [Q&A]
Whenever I launch OBS-Studio it does not show anything while works smoothly on GNOME
I am a newbie and cannot understand this?

**Comment by kRHYME7** [comment]
Hi! Please open this issue to the Compositor.

https://github.com/hyprwm/Hyprland

We are just a mere configuration repository. If anything in this repo is causing it please report it back here. For now we can't do anything related to that level.
